test/e2e/autoscaling/autoscaling_timer.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
you can use the opportunity to add people as reviewers and/or owners in this folder 
https://github.com/kubernetes/kubernetes/blob/9dbdfe7ecb16887c8495747303db7e1c2466c76f/test/e2e/autoscaling/OWNERS in addition to the alias, to have more granularity on the permissions

Reply:
Let's leave this as follow up @aojea ...